drivers/qwiic/micro_oled.c

```diff
@@ -374,13 +374,13 @@ if (LCDHEIGHT > 32) {
 void send_command(uint8_t command) {
   micro_oled_transfer_buffer[0] = I2C_COMMAND;
   micro_oled_transfer_buffer[1] = command;
-  i2c_transmit(I2C_ADDRESS_SA0_1 << 1, micro_oled_transfer_buffer, 2, 100);
+  i2c_transmit(I2C_ADDRESS_SA0_0 << 1, micro_oled_transfer_buffer, 2, 100);
```

Comment:
Is this correct? 

@jackhumbert @ishtob

Reply:
Hadron v3 has this piece of code in config.h:
```
#undef I2C_ADDRESS_SA0_1
#define I2C_ADDRESS_SA0_1 0b0111100
```

Perhaps we should make a new #define which sets either I2C_ADDRESS_SA0_1 or I2C_ADDRESS_SA0_0 in config.h.

most of the time, `0b0111100`will be correct if someone is using ssd1306 on a daughterboard, but if they are not using the daughterboard, it could be `0b0111101``
If we had to pick one for default, I'd go with `I2C_ADDRESS_SA0_0` i.e. 0b0111100